mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -216,3 +216,38 @@ Feature: Loans tests
     Then status 200
     Then match $.loans[0].dueDateChangedByRecall == true
     And match $.loans[0].dueDate !contains expectedDueDateBeforeRequest
+
+  Scenario: When an loaned item is checked in at a service point that serves its location and no request exists, change the item status to Available
+
+    * def extItemBarcode = '' + random(100000)
```

Comment:
Is it necessary to use random values instead of constants?

Reply:
not necessary of course, but as far as I know, Viktor suggested using random values in previous reviews. So, what should I do?

Reply:
@alexanderkurash done